README.md

```diff
@@ -244,40 +244,33 @@ npm run serve
 
 After the development server starts you have to navigate to `http://localhost:8080`, in order to use the Raiden dApp. It requires either MetaMask to be installed on your browser or some other web3 provider (e.g. Wallet apps with dApp support).
 
-## Use the dApp In Different Environments
+## Use the dApp in Different Environments
+
+If you want to continue using the dApp with a specific account when switching browser or changing computer you need to make sure that the dApp is connecting with the most recent state.
```

Comment:
Store a backup of your state **to avoid loosing tokens**, if you
- Delete your local browser storage
- Change your computer
- Change your browser